IBM/ibm-common-service-webhook
# removed podpreset webhook

because podpreset is no longer needed to work around a latency bug related to DNS that existed in OCP 3.x https://access.redhat.com/solutions/3827421.

How to test:

1. Deploy Bedrock LTSR
2. Create OperandRequest for `ibm-cert-manager-operator`
3. Note in the deployment YAML there is a DNS configuration as shown below
4. Build cs-webhook image from this PR with `make push-image`. Preferably run `export REGISTRY=quay.io/<username>` before doing so
5. Update the cs-webhook deployment YAML to use the PR image. Optionally use mine `quay.io/henry_h_li/ibm-cs-webhook-amd64:c97da14c-dirty`
6. Delete the OperandRequest and re-create it
7. Note that the newly created cert-manager-operator deployment no longer has the DNS configuration

```
dnsConfig:
    options:
      - name: single-request-reopen
```
